explorer/nodewatch/tests/test_NetworkConnector.py

```diff
@@ -12,12 +13,12 @@ def __init__(self, endpoint, has_api=True):
 		self.name = endpoint
 		self.has_api = has_api
 		self.height = 0
-		self.finalized_height = 0
+		self.finalized_info = FinalizedInfo(0, 0, None, 0)
 
 
 def _assert_node_descriptor(descriptor, height, finalized_height=0):
 	assert height == descriptor.height
-	assert finalized_height == descriptor.finalized_height
+	assert finalized_height == descriptor.finalized_info.height
```

Comment:
do you want to add (at least one) test somewhere that finalized_info structure is populated correctly with all fields?